modules/nodes-scheduler-node-selectors-pod.adoc

```diff
@@ -9,23 +9,21 @@ You can use node selector labels on pods to control where the pod is scheduled.
 
 With node selectors, {product-title} schedules the pods on nodes that contain matching labels.
 
-You can add labels to a node or MachineConfig, but the labels will not persist if the node or machine goes down.
-Adding the label to the MachineSet ensures that new nodes or machines will have the label.
-
 To add node selectors to an existing pod, add a node selector to the controlling object for that node, such as
 a ReplicaSet, Daemonset, or StatefulSet. Any existing pods under that controlling object are recreated on a node
 with a matching label. If you are creating a new pod, you can add the node selector directly
 to the pod spec.
 
+You can add labels to a node or MachineConfig, but the labels will not persist if the node or machine goes down.
+Adding the label to the MachineSet ensures that new nodes or machines will have the label.
+
 [NOTE]
 ====
-You cannot add a node selector to an existing scheduled pod.
+You cannot add a node selector directly to an existing scheduled pod.
 ====
 
-.Prerequisites
-
-If you want to add a node selector to existing pods, determine the controlling object for that pod.
-For exeample, the `router-default-66d5cf9464-m2g75` pod is controlled by the `router-default-66d5cf9464`
+To add a node selector to existing pods, determine the controlling object for that pod.
```

Comment:
I'd make this process a step instead of a prereq.

Reply:
It actually was a prereq.....